diff -up st-0.8.2-mod/x.c st-0.8.2/x.c
--- st-0.8.2-mod/x.c	2019-06-17 02:59:18.198965995 +0530
+++ st-0.8.2/x.c	2019-06-17 02:59:54.208320871 +0530
@@ -708,7 +708,7 @@ xloadcolor(int i, const char *name, Colo
 	XRenderColor color = { .alpha = 0xffff };
 
 	if (!name) {
-		if (BETWEEN(i, 16, 255)) { /* 256 color */
+		if (BETWEEN(i, 22, 255)) { /* 256 color */
 			if (i < 6*6*6+16) { /* same colors as xterm */
 				color.red   = sixd_to_16bit( ((i-16)/36)%6 );
 				color.green = sixd_to_16bit( ((i-16)/6) %6 );
